Wireframe/index.html

```diff
@@ -27,11 +27,11 @@ <h2>README file</h2>
         <p>
           
         </p>
-        <a href="">Read more</a>
+        <a href="https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-readmes">Read more</a>
```

Comment:
If you're using internal website links in your website, I suggest you to use` target="_blank"` in a  tag as well, which will allow user to open link in another window